CONTRIBUTING.md.tpl

```diff
@@ -0,0 +1,9 @@
+# Contributing to Scipian
```

Comment:
Will there be other repo templates like this? Do we want to store them all at the root of this repo or create a templates directory?

Reply:
Good point, yes there will be others. I like the idea of creating a directory to hold them.